ca.victorz.acr.AssaultCubeReloaded.yml

```diff
@@ -0,0 +1,52 @@
+app-id: ca.victorz.acr.AssaultCubeReloaded
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: client
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.assaultcube-reloaded
+
+modules:
+  - shared-modules/SDL/sdl12-compat.json
+  - shared-modules/SDL/SDL_image-1.2.12.json
+
+  - name: acreloaded
+    buildsystem: simple
+    subdir: source/src
+    sources:
+      - type: git
+        url: https://github.com/acreloaded/acr.git
+        tag: v2.18.2
+        commit: 86dd4b8911121efac1790516ef5bae14db687501
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+      - type: shell
+        commands:
+          - sed -i 's#CUBE_DIR=$(dirname "$(readlink -f "${0}")")#CUBE_DIR=/app/share/acreloaded/#g' client.sh
+          - sed -i 's#CUBE_OPTIONS="--home=home#CUBE_OPTIONS="--home=${HOME}/.assaultcube-reloaded#g' client.sh
+          - sed -i 's#CUBE_DIR=./#CUBE_DIR=/app/share/acreloaded/#g' server.sh
```

Comment:
a patch would be better here. `sed` is obscure and has no failure mode.

Reply:
@Mailaender this is unresolved

Reply:
Thank you for the suggestion, but I don't want to use `patch` here.

Reply:
so what happen when you decide to not maintain this anymore? Someone would need to figure out the obscurity of sed? Or why the package suddenly start to fail because sed has no failure mode ? This should be a patch. No necessarily upstreamable. But this should be a patch.

Reply:
I would like to keep this inline with the [Assault Cube manifest](https://github.com/flathub/net.cubers.assault.AssaultCube/blob/master/net.cubers.assault.AssaultCube.yml).

Reply:
easy: update it too.